src/lib/nodes/abstract.node.ts

```diff
@@ -97,13 +113,16 @@ export abstract class Node<C = unknown> {
   healthcheckInProgress = false
 
   constructor(
-    url: string,
+    endpoint: { alt_ip?: string; url: string },
```

Comment:
You can use the `NodeInfo` type from `src/types/wallets/index.ts`.

![image](https://github.com/user-attachments/assets/727ee4bd-5ce5-432d-8784-10ee31eb4ee7)

Reply:
Check other places as well